### DISCUSSION TITLE
Separate pinmux for boot and jernel

### DISCUSSION THREAD

**Discussion by albertbu** [OE4T/meta-tegra#1627] [General]
HI, 
I am trying to split our C7 PCI interface. So far, we are bb_append-ing the new pinmux files to the tegra-bootfiles, but then the system isn't even able to compete the recovery boot. 
Any instructions about correct placement of those files? 
In the "standard" Linux_for_Tegra, we had the bootloader/generic/BCT directory and the bootloader directory, but I can't figure it out how this is managed in the yocto layer. Thanks for helping!

**Comment by madisongh** [comment]
The specifics vary from model to model of Jetson module, and from version to version of L4T, but there's some documentation about this on [a wiki page](https://github.com/OE4T/meta-tegra/wiki/Creating-a-custom-MACHINE) here.
